docs/setup-guide.md

```diff
@@ -0,0 +1,107 @@
+# Guide to create scratch Org and stores
+
+## Pre-requisites
+
+1. A Salesforce Org where you want to create scratch Orgs and stores
+2. Enable Dev Hub on the Org
+    1. Navigate to Setup (Gear icon located in upper right corner)
+    2. Search for or navigate to Dev Hub
+    3. Switch Enable Dev Hub to ‘Enabled’
+
+## Setup
+
+1. Install SFDX CLI, see [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/) to learn more about the Salesforce CLI
+    ```bash
+    npm install --global sfdx-cli
+    ```
+2. Install commerce plugins
+    ```bash
+    sfdx plugins:install @salesforce/commerce
+    ```
+3. Install shane plugins
+    ```bash
+     sfdx plugins:install shane-sfdx-plugins
+    ```
+4. Verify sfdx-cli version
+
+    ```bash
+    sfdx -v
+
+    # sample output
+    sfdx-cli/7.182.1 darwin-arm64 node-v16.17.1
+    ```
+
+5. Verify sfdx plugin versions
+
+    ```bash
+    sfdx plugins
+
+    # sample output
+    @salesforce/commerce 242.0.26
+    shane-sfdx-plugins 4.43.0
+    ├─ @mshanemc/sfdx-sosl 1.1.0
+    └─ @mshanemc/plugin-streaming 1.1.7
+    ```
+
+6. Authorize your org with CLI
+
+    ```bash
+    sfdx force:auth:web:login -r <<INSTANCE_URL>> -a <<ORG_ALIAS>>
+
+    #####example#####
+    # sfdx force:auth:web:login -r https://login.test1.pc-rnd.salesforce.com -a mydevhub
+
+    ###### If you created a connected app ######
```

Comment:
Do we want to remove this example step?

Reply:
Just left it there in case anyone uses connected app